update teststate/.../system.sql

## Motivation

**Current architecture**

test state uses `state/teststate/testsys/system.sql` file

- **Critical Problem (hot)**: fiscalcloud test can not work:
```
--- FAIL: Test_Command_AddFiscalContainer (0.04s)
panic: system.sql:5:10: unexpected token "TABLE" (expected "WORKSPACE" <ident> ("INHERITS" DefQName ("," DefQName)*)? "(" ("DESCRIPTOR" WsDescriptorStmt)? WorkspaceStatement? (";" WorkspaceStatement)* ";"? ")")
	no valid schema files [recovered]
	panic: system.sql:5:10: unexpected token "TABLE" (expected "WORKSPACE" <ident> ("INHERITS" DefQName ("," DefQName)*)? "(" ("DESCRIPTOR" WsDescriptorStmt)? WorkspaceStatement? (";" WorkspaceStatement)* ";"? ")")
	no valid schema files

goroutine 7 [running]:
testing.tRunner.func1.2({0xce18e0, 0xc000009890})
	C:/Program Files/Go/src/testing/testing.go:1632 +0x225
testing.tRunner.func1()
	C:/Program Files/Go/src/testing/testing.go:1635 +0x359
panic({0xce18e0?, 0xc000009890?})
	C:/Program Files/Go/src/runtime/panic.go:785 +0x132
github.com/voedger/voedger/pkg/state/teststate.(*testState).buildAppDef(0xc000002300, {0xdb8711, 0x2e}, {0xd9d335?, 0x8b32a0?}, {0xc00059df48, 0x1, 0xc00018e0f0?})
	c:/workspace/go/voedger/pkg/state/teststate/impl.go:321 +0xc54
github.com/voedger/voedger/pkg/state/teststate.NewTestState(0x1, {0xdb8711, 0x2e}, {0xc00006ff48, 0x1, 0x1})
	c:/workspace/go/voedger/pkg/state/teststate/impl.go:73 +0x109
github.com/voedger/voedger/pkg/exttinygo/exttinygotests.NewTestAPI(0xc00006ff18?, {0xdb8711?, 0x81b9a0?}, {0xc00006ff48?, 0x0?, 0x604?})
	c:/workspace/go/voedger/pkg/exttinygo/exttinygotests/impl.go:17 +0x25
github.com/untillpro/airs-bp3/apps/fiscalcloud/wasm.Test_Command_AddFiscalContainer(0xc000113860)
	c:/workspace/go/airs-bp3/apps/fiscalcloud/wasm/impl_addcontainers_test.go:27 +0x67
testing.tRunner(0xc000113860, 0xde6590)
	C:/Program Files/Go/src/testing/testing.go:1690 +0xcb
created by testing.(*T).Run in goroutine 1
	C:/Program Files/Go/src/testing/testing.go:1743 +0x377
FAIL	github.com/untillpro/airs-bp3/apps/fiscalcloud/wasm	0.112s

```

## Technical design

- rename to .vsql
- put all table declarations into workspace

--- a/pkg/state/teststate/testsys/system.sql
+++ b/pkg/state/teststate/testsys/system.sql
@@ -1,29 +1,21 @@
 -- Copyright (c) 2020-present unTill Pro, Ltd.
 
--- note: this schema is for tests only. Voedger sys package uses copy of this schema
+-- note: this schema is for tests only
 
-ABSTRACT TABLE CRecord();
-ABSTRACT TABLE WRecord();
-ABSTRACT TABLE ORecord();
-ABSTRACT TABLE CDoc INHERITS CRecord();
-ABSTRACT TABLE ODoc INHERITS ORecord();
-ABSTRACT TABLE WDoc INHERITS WRecord();
-ABSTRACT TABLE CSingleton INHERITS CDoc();
 ABSTRACT WORKSPACE Workspace(
-	TABLE ChildWorkspace INHERITS CDoc (
-		WSName varchar NOT NULL,
-		WSKind qname NOT NULL,
-		WSKindInitializationData varchar(1024),
-		TemplateName varchar,
-		TemplateParams varchar(1024),
-		WSClusterID int32 NOT NULL,
-		WSID int64,           -- to be updated afterwards
-		WSError varchar(1024) -- to be updated afterwards
-	);
-);
-ALTERABLE WORKSPACE Profile();
+	ABSTRACT TABLE CRecord();
+	ABSTRACT TABLE WRecord();
+	ABSTRACT TABLE ORecord();
+	ABSTRACT TABLE CDoc INHERITS CRecord();
+	ABSTRACT TABLE ODoc INHERITS ORecord();
+	ABSTRACT TABLE WDoc INHERITS WRecord();
+	ABSTRACT TABLE CSingleton INHERITS CDoc();
 
-TABLE WorkspaceDescriptor INHERITS CSingleton (
+	TYPE Raw(
+		Body varchar(65535)
+	);
+
+	TABLE WorkspaceDescriptor INHERITS CSingleton (
 		-- owner* fields made non-required for app workspaces
 		OwnerWSID int64,
 		OwnerQName qname, -- Deprecated: use OwnerQName2
@@ -42,11 +34,21 @@ TABLE WorkspaceDescriptor INHERITS CSingleton (
 		InitCompletedAtMs int64,
 		Status int32,
 		OwnerQName2 text
+	);
+
+	TABLE ChildWorkspace INHERITS CDoc (
+		WSName varchar NOT NULL,
+		WSKind qname NOT NULL,
+		WSKindInitializationData varchar(1024),
+		TemplateName varchar,
+		TemplateParams varchar(1024),
+		WSClusterID int32 NOT NULL,
+		WSID int64,           -- to be updated afterwards
+		WSError varchar(1024) -- to be updated afterwards
+	);
 );
 
-TYPE Raw(
-    Body   varchar(65535)
-);
+ALTERABLE WORKSPACE Profile();
 
 EXTENSION ENGINE BUILTIN (
 
@@ -274,7 +276,7 @@ EXTENSION ENGINE BUILTIN (
 
 	STORAGE Logger(
 		/*
-		Key: 
+		Key:
 			LogLevel int32
 		Value
 			Message text

